Dockerfile

```diff
@@ -1,32 +1,33 @@
-FROM jupyter/base-notebook:27ba57364579
+FROM jupyter/base-notebook:814ef10d64fb
```

Comment:
"Updating" to jupyterhub 1.0.0 happens here, where they built a newer single user baseimage with the newest version of Jupyterhub. This one is based on the latest Z2J repo version here: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/tree/master/images/singleuser-sample